File: JAVA/8.Nested_For_1/p6.java
import java.util.*;

class Demo{

	public static void main(String[] args){
	
		Scanner sc = new Scanner(System.in);
		System.out.println("Enter no. of rows: ");
		int row = sc.nextInt();
		for(int i=0;i<row;i++){
		
			for(int j=0;j<row;j++){
			
				System.out.print((j+1)+"\t");
			}System.out.println();
		}
	}
}

File: Assignments/Assignment_13/p6.c
#include<stdio.h>
void main(){

        int row;
        printf("Enter the no of rows : \n");
        scanf("%d",&row);
        for(int i=row;i>0;i--){

                for(int j=i;j>0;j--){

                        printf("%d\t",i);
                }printf("\n\n");
        }
}
